### DISCUSSION TITLE
RDB ORM now supports Oracle

### DISCUSSION THREAD

**Discussion by lroal** [oracle/node-oracledb#1640] [Show and tell]
I am excited to announce that [RDB orm](https://github.com/alfateam/rdb) now supports Oracle.
It currenly only supports timestamp - and not date. Just let me know if this is an issue, and I will fix it.
RDB has full typescript intellisense.
Requires no code generation.
Supports both ESM and CJS. 
You can use with plain javascript as well.

**Comment by cjbj** [comment]
@lroal  Thanks for sharing!
